src/controllers/room.controller.js

```diff
@@ -0,0 +1,81 @@
+import { Room } from '../models/Room.models.js';
+
+export const getRooms = async (req, res) => {
+  res.send(await Room.findAll());
+};
+
+export const getOneRoom = async (req, res) => {
+  const { id } = req.params;
+  const room = await Room.findByPk(id);
+
+  if (!room) {
+    res.status(404);
+
+    return;
+  }
+  res.status(200);
+  res.send(room);
+};
+
+export const createRoom = async (req, res) => {
+  const { roomName } = req.body;
+
+  if (!roomName) {
+    res.status(400);
+  }
```

Comment:
Similar to the previous comment, here you are setting the status to 400 but not sending any response. The client will be left waiting for a response. You should send a response with a message informing that the roomName is required.